### PR TITLE
Add blogpost for HLS 0.2.0

### PR DESCRIPTION
**Pull request by fendor**
I haven't tested the gif's display correctly, yet

### CODE REVIEW

File: ide/posts/2020-07-17-hls-update.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+-- >>> let evaluation = " evaluation"
+-- >>> unwords example ++ evaluation
+example :: [String]
+example = ["This","is","an","example","of"]
+```
+
+Executing these haddock code-examples by hand is a bit tedious. It is way easier to just execute a codelens and see the result.
+With the "Eval" plugin, it is as easy as a single click to produce the relevant output:
+
+[![Evaluate Haddock code comment](https://i.imgur.com/raaTFKr.gif)
```
Comment (lukel97) [inline ide/posts/2020-07-17-hls-update.md]:
```suggestion
![Evaluate Haddock code comment](https://i.imgur.com/raaTFKr.gif)
```

File: ide/posts/2020-07-17-hls-update.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
+---
+posttitle: Haskell Language Server 0.2.0
+author: Fendor
+date: 2020-07-17
+github: fendor
+---
+
+We are back for another update of this summer's Haskell IDE efforts.
+Quite some time has passed since our last update, so we have a lot to tell you about!
+We are going to speak about what is new in the latest release and what other new features are already waiting  in the pipeline.
```
Comment (lukel97) [inline ide/posts/2020-07-17-hls-update.md]:
```suggestion
We are going to speak about what is new in the latest release and what other new features are already waiting in the pipeline.
```

File: ide/posts/2020-07-17-hls-update.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+A natural extension of this will be to make it possible to download these binaries from your editor extension. This is also in the making, although, for now, only for the vscode extension `vscode-hie-server`.
+With prebuilt binaries, we hope to make the setup experience for newcomers easier to understand and faster to follow and no longer force users to compile everything from scratch which can take hours and hours.
+
+As the cherry on top, we plan to integrate these pre-built binaries with the successful tool [ghcup](https://www.haskell.org/ghcup/). This will improve the tooling story for Haskell and setting up from zero to a full-blown development environment will be a matter of minutes.
+
+### Simple Eval Plugin
+
+A new plugin called "Eval" will be added soon, the Haskell Language Server! Its purpose is to automatically execute code in haddock comments to make sure that example code is up-to-date to the actual implementation of the function. This does not replace a proper CI, nor [doctest](https://github.com/sol/doctest#readme), but it is a simple quality of life improvement!
```
Comment (pepeiborra) [inline ide/posts/2020-07-17-hls-update.md]:
```suggestion
A new plugin called "Eval" will be added soon to the Haskell Language Server! Its purpose is to automatically execute code in haddock comments to make sure that example code is up-to-date to the actual implementation of the function. This does not replace a proper CI, nor [doctest](https://github.com/sol/doctest#readme), but it is a simple quality of life improvement!
```

**Review by lukel97 — COMMENTED** [review]
LGTM other than those typos

File: ide/posts/2020-07-17-hls-update.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+import Data.List (unwords)
+
+-- >>> let evaluation = " evaluation"
+-- >>> unwords example ++ evaluation
+example :: [String]
+example = ["This","is","an","example","of"]
+```
+
+Executing these haddock code-examples by hand is a bit tedious. It is way easier to just execute a codelens and see the result.
```
Comment (lukel97) [inline ide/posts/2020-07-17-hls-update.md]: Nitpick
```suggestion
Executing these haddock code-examples by hand is a bit tedious. It is way easier to just execute a _code lens_ and see the result.
```

File: ide/posts/2020-07-17-hls-update.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+
+Nevertheless, I will tease some upcoming improvements, with a disclaimer, that we cannot promise that the features will make it into the next release.
+
+### Prebuilt Binaries
+
+This has been a long requested feature! The first issue I can find about it was created for Haskell IDE Engine in January 2019. Back then, Haskell IDE Engine was facing a number of road blocks, such as data-files that are not easily relocatable and reliance on packages such as `ghc-paths` which compile important run-time information into the binary. Piece by piece, these issues have been resolved by patching upstream libraries, using alternative APIs and querying the run-time information at, well, run-time. Major changes to [hie-bios](https://github.com/mpickering/hie-bios/) were necessary in order to make it possible to find the information we care about.
+
+Now we are close to being able to offer pre-built binaries for major platforms, such as windows, macos and (posix compliant) linux systems.
```
Comment (lukel97) [inline ide/posts/2020-07-17-hls-update.md]: Nitpick
```suggestion
Now we are close to being able to offer pre-built binaries for Windows, macOS and Linux.
```

File: ide/posts/2020-07-17-hls-update.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+A natural extension of this will be to make it possible to download these binaries from your editor extension. This is also in the making, although, for now, only for the vscode extension `vscode-hie-server`.
+With prebuilt binaries, we hope to make the setup experience for newcomers easier to understand and faster to follow and no longer force users to compile everything from scratch which can take hours and hours.
```
Comment (lukel97) [inline ide/posts/2020-07-17-hls-update.md]: Nitpick
```suggestion
With prebuilt binaries, we hope to make the setup experience for newcomers easier and faster, without the need to compile everything from scratch, which can take hours and hours.
```

**Pull request merged**